catalog/region-specific-policies.yaml

```diff
@@ -0,0 +1,22 @@
+- sid: "DenyRegionUsage"
+  effect: "Deny"
+  not_actions:
+    - "cloudfront:*"
+    - "iam:*"
+    - "route53:*"
+    - "support:*"
+    - "organizations:*"
+    - "waf:*"
+    - "budgets:*"
+    - "globalaccelerator:*"
+    - "cur:*"
+    - "ce:*"
+    - "directconnect:*"
+  condition:
+    - test: "StringNotEqualsIgnoreCase"
+      variable: "aws:RequestedRegion"
+      values:
+        - "${region1_lockdown}"
+        - "${region2_lockdown}"
```

Comment:
You may want to consider more descriptive names and also a comment to indicate what the meaning of these two variables is?